### ISSUE TITLE
Possible to make it look like it does in Atom?

### ISSUE THREAD

**Issue by jmangz** [file-icons/vscode#39]
Hi, I'm coming from Atom and I prefer the look of the file structure when it is Atom. Is there a way to make it look more like it here in VS Code? Comparing the two it seems like the icons are a bit larger in Atom.

**Comment by Alhadis** [comment]
@DanBrooker Do you mind if I take up duties on keeping this package up-to-date?

While I don't use VSCode as my editor, I don't fancy seeing this package decay and fall behind. Just asking out of courtesy since it's been entirely in your care so far... =)

**Comment by DanBrooker** [comment]
@Alhadis I don't mind at all, I'll figure out how to add you to the vsce publish.

**Comment by Alhadis** [comment]
Thanks! 😀

**Comment by DanBrooker** [comment]
@Alhadis do you have a non gmail address? https://marketplace.visualstudio.com doesn't think gmail.com is a valid domain.
<img width="260" alt="Screen Shot 2019-04-25 at 9 47 42 PM" src="https://user-images.githubusercontent.com/714197/56726965-e3c24280-67a3-11e9-8407-5227de2d0463.png">

**Comment by Alhadis** [comment]
Yeah. Just `s/gmail/hotmail/` and it'll be delivered to an address I haven't checked in years (and until now, have been afraid to do so). 😁

So in other words: gardnerjohng@hotmail.com

**Comment by DanBrooker** [comment]
@Alhadis same error, weird maybe try sign up here first https://marketplace.visualstudio.com/

**Comment by Alhadis** [comment]
Oh, right. It needed to be connected to my account or something.

I've signed into https://marketplace.visualstudio.com/, so... try now. :)

**Comment by DanBrooker** [comment]
Nope, I've submitted a support ticket

**Comment by Alhadis** [comment]
> Welcome to Azure DevOps

I have no idea what the hell that is. This is gonna be fun. 😀

lol, Micro$oft

**Comment by Alhadis** [comment]
BTW, I don't know if it's relevant, but I set my contact e-mail on that account to my gmail. It's aware of both accounts.

I really have no idea if that makes things easier or a thousand times harder...
